README.md

```diff
@@ -17,18 +17,18 @@ Other terms used below:
 A minimalistic and stateless version of dusk-blockchain node where Consensus protocol in conjuction with Kadcast is able to join and participate `dusk-blockchain`/test-harness. Useful for testing and diagnostic incompatibility issues. This executable should be deprecated once dusk-blockchain is fully migrated.
 
 ## Example Testbed
-A multi-instance setup running 10 SBA instances provisioned with 10 eligible participants. The setup is configured to run for up to 1000 rounds. Useful for any kind of testing (issues, stress and performance testing).
+A multi-instance setup running 10 SA instances provisioned with 10 eligible participants. The setup is configured to run for up to 1000 rounds. Useful for any kind of testing (issues, stress and performance testing).
 
 ## Consensus crate
-A full implementation of SBA mechanism.
+A full implementation of SA mechanism.
 
  # Implementation details 
-SBA is driven basically by two tokio-rs tasks - Main Loop and Agreement Loop. Each of them has its own inbound and outbound message queues/channels to exchange messages with outsiders. SBA protocol parameters are defined in `/src/config.rs`.
+SA is driven basically by two tokio-rs tasks - Main Loop and Agreement Loop. Each of them has its own inbound and outbound message queues/channels to exchange messages with outsiders. SA protocol parameters are defined in `/src/config.rs`.
 
 - `Main_Loop uses` `Operations trait` to execute Contract Storage calls and `Database trait` to store and retrieve candidate blocks. It is mainly responsible to execute Selection, 1th Reduction and 2nd Reduction steps in a row and eventually produce/broadcast an **Agreement Message**. Inbound queue for Main_Loop can contain messages of either **NewBlock** or **Reduction** type.
 
 
-- `Agreement_Loop` uses `Database trait` to retrieve candidate blocks when a winner hash is found. It is mainly responsible to verify and accumulate **Agreement messages** from different Consensus iterations and process **Aggregated Agreement message**. Inbound queue for Agreement_Loop can contain messages of either **Agreement** or **AggrAgreement** type. An Agreement message is verified and accumulated concurrently by a pool of workers/verifiers - again tokio-rs tasks.
+- `Agreement_Loop` uses `Database trait` to retrieve candidate block when a winner hash is found. It is mainly responsible to verify and accumulate **Agreement messages** from different Consensus iterations and process **Aggregated Agreement message**. Inbound queue for Agreement_Loop can contain messages of either **Agreement** or **AggrAgreement** type. An Agreement messages are verified and accumulated concurrently by a pool of workers/verifiers - again tokio-rs tasks.
```

Comment:
- a candidate block
- not sure on the verb 'found' (the block hashes are already known, it's just a matter of choosing). I'd say 'obtained' or 'selected' 
- Remove 'An' from last sentence

Reply:
Done.